flang/lib/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_subdirectory(Decimal)
 add_subdirectory(Lower)
 add_subdirectory(Parser)
 add_subdirectory(Semantics)
+add_subdirectory(Support)
```

Comment:
Mainly to match the layout in both `llvm` and `mlir`. In principle, it could be used in other tools, both in-tree and out-of-tree. It feels odd to have to link against the entire frontend just to get a timer.